spec/engine/miq_ae_method_spec.rb

```diff
@@ -47,6 +47,22 @@ def root
       end
     end
 
+    context "with a script that tries to load yaml with aliases" do
+      let(:script) do
+        <<-RUBY
+          YAML.load("---\na: &a\n  b: true  \n\ndevelopment:\n  <<: *a\n  c: false\n\n")
```

Comment:
Not really important, but to make the intention clearer, should we also have a test that calls safe_load (or perhaps just change this test to call safe_load)?  Up to you...`load` is more common, and we've overridden safe_load, so this does verify that `load` _also_ works in addition to `safe_load`

Reply:
yes, we should have both tests, good idea... interfaces of load and safe_load changed over time so I'm fine with being cautious with testing it